src/main/java/frc/robot/Robot.java

```diff
@@ -10,15 +10,16 @@
 import edu.wpi.first.wpilibj.TimedRobot;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.CommandScheduler;
+
 /**
  * The VM is configured to automatically run this class, and to call the functions corresponding to
  * each mode, as described in the TimedRobot documentation. If you change the name of this class or
  * the package after creating this project, you must also update the build.gradle file in the
  * project.
  */
 public class Robot extends TimedRobot {
-    public static boolean debug = false;
     public static final AHRS navx = new AHRS(SPI.Port.kMXP);
+    public static boolean debug = false;
```

Comment:
Why?

Reply:
Because we want to be able to change the value when we are debugging (no matter the time)